control-plane-operator/controllers/hostedcontrolplane/cvo/params.go

```diff
@@ -41,5 +42,15 @@ func NewCVOParams(hcp *hyperv1.HostedControlPlane, images map[string]string) *CV
 	p.DeploymentConfig.SetRestartAnnotation(hcp.ObjectMeta)
 	p.DeploymentConfig.SetControlPlaneIsolation(hcp)
 	p.DeploymentConfig.Replicas = 1
+
+	if explicitNonRootSecurityContext {
```

Comment:
Instead of doing this here, can we just modify DeploymentConfig to have a bool (ExplicitNonRootSecurityContext) and then when it's applied (as in https://github.com/openshift/hypershift/blob/main/support/config/deployment.go#L162-L185), we just update the security context for every container in the deployment?

Reply:
Great point. I went ahead and extended `DeploymentConfig` and added that bool...